.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,17 @@
 
-GitHub issue resolved: # <!-- add a Github issue number here, e.g #123. This line 
-                              automatically closes the issue when the PR is merged --> 
+GitHub issue resolved: # <!-- add a Github issue number here, e.g #123. -->
```

Comment:
I'm not 100% sure but I believe this may no longer work when using squash and merge because the PR description won't end up in the commit message. This is how auto-closing of issues worked in the past. It _might_ still work though based on whether the PR is actually linked to an issue (which is a feature Github introduced very recently). In any case, I'm taking it out for brevity - if it still works that's great, if not, we're not misleading people.